src/blk/BlockDevice.h

```diff
@@ -64,6 +64,14 @@
 #define  WRITE_LIFE_MAX  	1
 #endif
 
+enum struct FAMode {            // to be used in the buffered parameter
+  DIRECT = false,
```

Comment:
any reason to assign these enum to a given value?

Reply:
To be more clear about the values and the type of them .... 
And it immediatly tells you that there should only be 2 values be used: `true` and `false`
Otherwise it would be just 0 and 1 which is more or less the old C-style boolean way.
But somebody could be tempted to actually define something with a value of 2
I'm not particular on this, but I find it much more descriptive

Reply:
i'd prefer not assigning values to them unless it's necessary.